coretex/_task/_local.py

```diff
@@ -86,13 +88,22 @@ def onCleanUp(self) -> None:
 
 class LocalArgumentParser(Tap):
 
+    parameters: List[BaseParameter]
+
     username: Optional[str]
     password: Optional[str]
 
     projectId: int
     name: Optional[str]
     description: Optional[str]
 
+    def __init__(self, parameters: List[BaseParameter]) -> None:
+        self.parameters = parameters
+        for parameter in parameters:
+            setattr(self, parameter.name, None)
```

Comment:
Why is `setattr` used here?

Reply:
To dynamically create attributes for the LocalArgumentParser class, which are populated with parse_known_args, and then read with getattr in _getParsedParameterValue

Reply:
Add a comment which describes why was this approach used